addons/snailmail/__manifest__.py

```diff
@@ -23,6 +22,7 @@
     'auto_install': True,
     'assets': {
         'mail.assets_messaging': [
+            'snailmail/static/src/new/**/*.js',
```

Comment:
bundle structure should follow new code structure in mail (`mail.assets_backend`)